dijkstra/dijkstra.c

```diff
@@ -0,0 +1,65 @@
+/*
+*  Following implementation of dijsktra's algorithm prints the minimum distance from given source to destination using adjacency matrix.
+*  Time complexity : O(V^2).
+*  Space Complexity : O(V).
+*/
+
+#include <stdio.h>
+#include <limits.h>	// For using INT_MAX
+#include <stdbool.h>	// bool datatype
+#include <string.h>	// for using memset()
+
+#define VERTICES 8
```

Comment:
Okay that's a drawback of using `#define`. I'll make `VERTICES` as a variable and pass it to all functions then it can be used as is 😄